src/class/cdc/cdc.h

```diff
@@ -50,6 +50,10 @@
  extern "C" {
 #endif
 
+#pragma GCC diagnostic push

```

Comment:
did gcc issue any warnings for packed attribute. I didn't see it, maybe we should solve the warning instead of suppressing it

Reply:
@tannewt I still didn't see any gcc warning regarding packed locally, can you explain it a bit more